src/services/export.js

```diff
@@ -238,51 +419,121 @@ const saveEpub = async (page, interactive) => {
   } catch (authorErr) {
     console.error(authorErr);
   }
- */
-  // @TODO get background element
+  */
   // get background to use as cover
   let background = COVER_DEFAULT_PATH;
   try {
-    background = await page.$eval('div.header', getBackground, GRAASP_HOST);
+    background = await page.$eval(HEADER, getBackground, GRAASP_HOST);
     if (!(background instanceof String) && typeof background !== 'string') {
-      background = null;
+      background = COVER_DEFAULT_PATH;
     }
   } catch (err) {
     console.error(err);
   }
 
   // replace relative images with absolute
-  await page.$$eval('img', makeImageSourcesAbsolute, GRAASP_HOST);
+  await page.$$eval(IMAGES, makeImageSourcesAbsolute, GRAASP_HOST);
 
   // screenshot replacements have to come after image src changes
 
-  // replace gadgets
-  const gadgets = await page.$$('div.gadget');
+  // one file labs
+  const offlineIframes = await page.$x(OFFLINE_READY_IFRAMES);
+  let offlineIframesScreenshots = [];
```

Comment:
For consistency with the other screenshot array variables, variable should be called `offlineIframeScreenshots`.